### ISSUE TITLE
umxGxE support ordinal

### ISSUE THREAD

**Issue by tbates** [tbates/umx#126]
- [ ] Support the new means/covs framework for `top;MZ;DZ` style models.
- [ ] Support ordinal DVs

A formula interface for specifying the means residualisation would be nice. It would support residualizing means for both twin's values of moderator, but also allow more complex formulae, such as `DV ~ T1mod + T1mod^2 + T1mod^3 )`

**Comment by yildirimselen** [comment]
_umxGxE_ currently does not solve binary models. After recoding the trait as a factor variable, the error message tells me that selDV must be numeric. If I run the analysis without using the factor variable, I get a result which treats the dependent variable as a continues variable. It would be a nice addition if umxGxE supports binary traits.

**Comment by mcneale** [comment]
> _umxGxE_ currently does not solve binary models. 
Agreed, it would be nice if it did.  In the meantime, you could implement the model in OpenMx.  There may be some example scripts on the <a href="https://www.colorado.edu/ibg/international-workshop/2020-international-statistical-genetics-workshop/workshop-2020-file">Boulder workshop website</a>.  Also, see https://vipbg.vcu.edu/vipbg/Articles/BG-2009-39_NoteParaEModelBinData.pdf for special considerations with moderated binary/ordinal variables.  There are classic Mx scripts in the appendix of that article, which might be a good place to start an OpenMx or umx contributed port.

**Comment by tbates** [comment]
OK. This looks like a pretty easy fix.
Will implement support for ordinal DV in `umxGxE`

**Comment by tbates** [comment]
Done.
